specs/projectreuniondeployment/reuniondeployment.md

```diff
@@ -0,0 +1,106 @@
+
+# Background
+
+This is the spec for proposal [Project Reunion Deployment Information API #155](https://github.com/microsoft/ProjectReunion/issues/https://github.com/microsoft/ProjectReunion/issues/155).
+
+Reunion consists of a Main package and framework published by Microsoft while consumers of Reunion will often be non-Microsoft publishers. In order to have Reunion fully present and functional, the Reunion Framework and Main packages must both be deployed, and depending on the operating system, OS updates may also be required.  While the Windows Application Model naturally supports framework dependencies, it does not have a means of declaring a dependency across Main packages of different publishers. This API spec and feature proposal is to provide a means of identifying when the Reunion dependencies are not satisfied, and if that is the case, provide URIs for where the missing dependencies could be satisifed by the user and/or the developer.
```

Comment:
Project Reunion. Never say just Reunion

Reply:
Fixed everywhere, including the filename.